java/change-notes/2021-06-01-statement-toString.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The CodeQL predicate `toString()` has been overridden for subclasses of `Stmt` to be more descriptive. Due to this, custom queries and `.expected` files of CodeQL tests for custom queries might have to be adjusted to match the new `toString()` results.
```

Comment:
I don't think we expect too much dependence on this change, so I think we should reduce it to just:
```suggestion
* The CodeQL predicate `toString()` has been overridden for subclasses of `Stmt` to be more descriptive.
```